tests/functional/api/moderation_test.py

```diff
@@ -108,6 +108,13 @@ def user(db_session, factories):
     return user
 
 
+@pytest.fixture
+def other_user(db_session, factories):
```

Comment:
This didn't exist before so the annotation.userid pointed to a non-existing user.